opentelemetry-sdk/src/opentelemetry/sdk/_logs/__init__.py

```diff
@@ -318,9 +320,29 @@ def __init__(
 
     @staticmethod
     def _get_attributes(record: logging.LogRecord) -> Attributes:
-        return {
+        attributes = {
             k: v for k, v in vars(record).items() if k not in _RESERVED_ATTRS
         }
+        if record.exc_info:
+            exctype, value, tb = record.exc_info
+            if exctype is not None:
+                exc_type = exctype.__name__
+            if value is not None:
+                message = value.args[0]
+            callstack = []
+            if tb is not None:
+                for fileName, line, method, text in traceback.extract_tb(tb):
+                    callstack.append({
```

Comment:
The format of the stacktrace can be discussed. Using this for now because it looks "nice".